### PR TITLE
chore: update bot sso readme to v5.0.0

### PR DESCRIPTION
**Pull request by yukun-dong**
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/17554194/

### CODE REVIEW

File: bot-sso/README.md [hunk 1]
```diff
@@ -1,79 +1,55 @@
...
+    1. Open the `Run and Debug Activity` Panel. Select `Launch Remote (Edge)` or `Launch Remote (Chrome)` from the launch configuration drop-down.
+- From TeamsFx CLI:
+    1. Run command: `teamsfx preview --env dev`.
+
+
+
+## Version History
+|Date| Author| Comments|
+|---|---|---|
+|Dec 15, 2022| zhany | update to support Teams Toolkit V4.2.0|
```
Comment (summzhan) [inline bot-sso/README.md:50]: The first two rows in this table is just fake content in templates, remove it and replace it with real version history.

**Review by summzhan — CHANGES_REQUESTED** [review]
one comment left in file.

File: bot-sso/README.md (hunk 1 above)
Comment (yukun-dong) [reply bot-sso/README.md]: Thanks. These 2 rows are removed.

**Comment by yukun-dong** [comment]
wrong branch, I will create another PR.

**Pull request closed without merging**